project/src/main/java/steps/LoginSteps.java

```diff
@@ -67,4 +85,10 @@ public void loginEvenIfLoggedId(UserCredentials user) {
 		}
 		loginWithValidCredentials(user);
 	}
+
+	@Step("Verify that password field is empty")
+	public void assertThatPasswordFieldEmpty() {
+		String currentPasswordValue = loginActions.getCurrentPasswordValue();
+		Assert.assertThat("Password field should be empty", currentPasswordValue.length(), is(0));
```

Comment:
Yes, makes sense, it's an accidental leftover from the master-class. I will remove it, and we'll add an implementation separately with better accuracy.